migrations/main.sql

```diff
@@ -530,3 +529,71 @@ CREATE VIEW user_request_list_view AS
     ORDER BY
         ur.university_id,
         ur.user_id;
+
+
+-- Create table commandant
+CREATE TABLE IF NOT EXISTS commandant(
+    commandant_id integer,
+    full_name VARCHAR(255),
+    telephone_number varchar(255) NOT NULL UNIQUE,
+    CONSTRAINT commandant_pk PRIMARY KEY(commandant_id));
+
+-- Insert data to table commandant
+INSERT INTO commandant(commandant_id, full_name, telephone_number) VALUES (1, 'Ляшко Надія Михайлівна', '380577107851');
+INSERT INTO commandant(commandant_id, full_name, telephone_number) VALUES (2, 'Любченко Володимир Віталійович', '380577792654');
+INSERT INTO commandant(commandant_id, full_name, telephone_number) VALUES (3, 'Колосова Олена Іванівна', '380573368350');
+INSERT INTO commandant(commandant_id, full_name, telephone_number) VALUES (4, 'Марченко Тетяна Федорівна', '380573367757');
+INSERT INTO commandant(commandant_id, full_name, telephone_number) VALUES (5, 'Рилова Лариса Миколаївна', '380577021194');
+INSERT INTO commandant(commandant_id, full_name, telephone_number) VALUES (6, 'Голубєва Надія Олександрівна', '380573401082');
+INSERT INTO commandant(commandant_id, full_name, telephone_number) VALUES (7, 'Піпенко Світлана Миколаївна', '380573910283');
+
+-- Create table hostel
+CREATE TABLE IF NOT EXISTS hostel(
+    hostel_id integer,
+    university_id integer,
+    number integer,
+    name VARCHAR(50),
+    city VARCHAR(50),
+    street VARCHAR(100),
+    build VARCHAR(5), 
+    commandant_id integer,
+    CONSTRAINT hostel_pk PRIMARY KEY(hostel_id));
+
+-- link comandant_id from table hostel as FK to commandant.commandant_id
+ALTER TABLE hostel ADD CONSTRAINT hostel_commandant_fk
+FOREIGN KEY (commandant_id) REFERENCES commandant(commandant_id) 
+MATCH FULL ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
Are you sure that it's right logic? If we delete commandant, hostel will be deleted too